django/contrib/auth/tokens.py

```diff
@@ -59,18 +58,10 @@ def _make_token_with_timestamp(self, user, timestamp):
 
         hash = salted_hmac(
             self.key_salt,
-            self._make_hash_value(user, timestamp),
+            user.get_hash_value(timestamp),
```

Comment:
I'd leave this method in and attach `timestamp` here -- there is no reason to pass this down to the user model